pubspec.yaml

```diff
@@ -1,15 +1,19 @@
 name: http
-version: 0.12.2
+version: 0.12.3-nullsafety
 homepage: https://github.com/dart-lang/http
 description: A composable, multi-platform, Future-based API for HTTP requests.
 
 environment:
-  sdk: ">=2.4.0 <3.0.0"
+  sdk: '>=2.10.0-0 <2.11.0'
 
```

Comment:
```suggestion

# Cannot be published until null safety ships
publish_to: none
```

Reply:
We're going to HAVE to publish this, I think, to enable migration of flutter gallery – or are we going to just use git deps?

Reply:
Please add the `publish_to: none` until we have a min SDK version that adds it to the allow list if we need to publish.

Publishing with a lower bound of `2.10.0-0` is not safe, and we need to prevent it from happening.

Reply:
done!

Should I change the lower bound?

Reply:
No, there is no lower bound at which it is safe to publish until we have a known SDK version that supports it so keeping it at `2.10.0-0` is fine.